src/Plugin/Field/FieldType/AutomatedListingConfiguration.php

```diff
@@ -95,6 +97,36 @@ public function storageSettingsForm(array &$form, FormStateInterface $form_state
       '#disabled' => $has_data,
     ];
 
+    $expose_filter_operator_default_value = $this->getSetting('expose_filter_operator');
+    if (!isset($options[$expose_filter_operator_default_value])) {
+      $expose_filter_operator_default_value = 0;
+    }
+
+    $element['expose_filter_operator'] = [
+      '#type' => 'radios',
+      '#title' => t('Expose Filter Operator'),
+      '#options' => [0 => 'Hide', 1 => 'Show'],
+      '#default_value' => $expose_filter_operator_default_value,
+      '#required' => TRUE,
+      '#weight' => -10,
+      '#disabled' => $has_data,
```

Comment:
hi @asarfara,

It means that once a paragraph is created, you are no longer able to change this setting.

Reply:
Yep will update it :)